pkg/costmodel/intervals_test.go

```diff
@@ -150,39 +151,47 @@ func TestGetIntervalPointsFromWindows(t *testing.T) {
 }
 
 func TestGetPVCCostCoefficients(t *testing.T) {
+	pod1Key := newPodKey("cluster1", "namespace1", "pod1")
```

Comment:
Tests have been audited and updated and augmented.

Reply:
These look great - thanks for adding :)